daemons/attrd/attrd_ipc.c

```diff
@@ -153,9 +150,8 @@ attrd_client_clear_failure(pcmk__request_t *request)
         crm_xml_replace(xml, PCMK__XA_ATTR_VALUE, NULL);
     }
 
-    attrd_client_update(xml);
     pcmk__set_result(&request->result, CRM_EX_OK, PCMK_EXEC_DONE, NULL);
```

Comment:
Do we still need this now that attrd_client_update() does it?

Reply:
Nope, definitely not.